src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -328,19 +327,20 @@ export default class AircraftCommander {
      * @return {array} [success of operation, readback]
      */
     runHold(aircraft, data) {
-        const turnDirection = data[0];
-        const legLength = data[1];
-        const holdFix = data[2];
```

Comment:
we should keep these re-assignments because it adds additional context to otherwise anonymous variables.  `data[2]` means nothing whereas `holdFix` does.

Reply:
done